nixos/modules/programs/skim.nix

```diff
@@ -8,38 +8,30 @@ let
   inherit (lib)
     mkEnableOption
     mkPackageOption
-    optional
     optionalString
+    mkIf
     ;
   cfg = config.programs.skim;
 in
 {
   options = {
     programs.skim = {
-      fuzzyCompletion = mkEnableOption "fuzzy completion with skim";
+      enable = mkEnableOption "Whether to enable the skim fuzzy finder.";
```

Comment:
`mkEnableOption` can be a bit unintuitive, it already adds this part of the description

https://github.com/NixOS/nixpkgs/blob/8283aacd75f646355c20670fae175e81fc77a700/lib/options.nix#L191

With this change, the docs would render as "Whether to enable Whether to enable ...."